packages/happy-dom/src/nodes/html-element/HTMLElement.ts

```diff
@@ -383,9 +383,13 @@ export default class HTMLElement extends Element implements IHTMLElement {
 
 		this.ownerDocument['_activeElement'] = null;
 
-		for (const eventType of ['blur', 'focusout']) {
+		const events = <const>[
+			['blur', false],
+			['focusout', true]
+		];
+		for (const [eventType, bubbles] of events) {
```

Comment:
I think we can replace the for loop logic with just (the same for the other for loop):
```javascript
this.dispatchEvent(new FocusEvent('blur', {
	bubbles: false,
	composed: true
}));
this.dispatchEvent(new FocusEvent('focusout', {
	bubbles: true,
	composed: true
}));
```

Settings `event._target` and `event._currentTarget` doesn't seem to do anything as it will be set by `dispatchEvent()` anyway.

Reply:
Sounds good. Updated